### PR TITLE
Registration Feature, Database config

### PR DESCRIPTION
**Pull request by Jbrooks00**
Some merge conflicts with angular dependencies, resolved. 
Back end and front end registration features implemented

### CODE REVIEW

**Review by Anth0nyQ — APPROVED** [review]
Ignore this comment, see the real one down below.

File: angular.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "$schema": "../node_modules/@angular/cli/lib/config/schema.json",
```
Comment (Anth0nyQ) [inline angular.json:2]: Required an additional dot to indicate that the actual file is a folder farther up, good work.

File: package.json.orig [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "go-mechanic",
```
Comment (Anth0nyQ) [inline package.json.orig:2]: This is just an initial file that was required for the start-up of our project.

**Review by Anth0nyQ — APPROVED** [review]
Viewed all pages of code and confirmed it all functions as intended, thanks Joseph!

**Comment by Jbrooks00** [comment]
Furthermore, the navbar logo was added, updated navbar components and maintenance pages

**Pull request merged**